lib/sdr_client/deposit.rb

```diff
@@ -10,8 +10,14 @@ def self.run(label: nil,
                  catkey:,
                  source_id:,
                  url:, files: [])
-      Process.new(label: label, type: type, url: url, files: files,
-                  apo: apo, collection: collection, catkey: catkey, source_id: source_id).run
+
+      metadata = Request.new(label: label,
```

Comment:
Is this an HTTP Request? If so, seems like an odd way to be encapsulating method arguments.

Reply:
no, it's a data model that represents a request in the abstract sense